Source/Microsoft.Gateway/Microsoft.InnerEye.Azure.Segmentation.Client/InnerEyeSegmentationClient.cs

```diff
@@ -152,19 +150,17 @@ public sealed class InnerEyeSegmentationClient : IInnerEyeSegmentationClient
         /// Initializes a new instance of the <see cref="InnerEyeSegmentationClient"/> class.
         /// </summary>
         /// <param name="baseAddress">The base address.</param>
-        /// <param name="licenseKeyEnvVar">The license key environment variable.</param>
+        /// <param name="licenseKey">The license key.</param>
         public InnerEyeSegmentationClient(
             Uri baseAddress,
-            string licenseKeyEnvVar)
+            string licenseKey)
```

Comment:
Now that's it's clear that this is the actual license key, may pass around a SecureString?

Reply:
I didn't know there was a SecureString! I'll do that

Reply:
Are we doing this?

Reply:
No. The official docs for SecureString don't recommend it